src/read/line.rs

```diff
@@ -601,6 +601,9 @@ fn line_change(i: &str) -> IResult<Change> {
     let (value, target) = cut(alt((
         map(tag("yaw"), |_| ChangeTarget::Yaw),
         map(tag("pitch"), |_| ChangeTarget::Pitch),
+        map(tag("target_yaw_offset"), |_| {
```

Comment:
Since you're adding new syntax, you should add tests. Check what I did in past commits.

Reply:
Hmmm, there are no tests for other `line_change` call. Not sure if I should follow that as well.